README.md

```diff
@@ -1,3 +1,37 @@
-# Jellyfin Chromecast
+<h1 align="center">Jellyfin Chromecast Client</h1>
+<h3 align="center">Part of the <a href="https://jellyfin.org">Jellyfin Project</a></h3>
 
-Source code for the Jellyfin Chromecast receiver application
+<p align="center">
+<img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
+<br/>
+<br/>
+<a href="https://github.com/jellyfin/jellyfin">
+<img alt="GPL 2.0 License" src="https://img.shields.io/github/license/jellyfin/jellyfin-chromecast.svg"/>
+</a>
+<a href="https://github.com/jellyfin/jellyfin/releases">
+<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-chromecast.svg"/>
+</a>
+</p>
+
+Jellyfin chromecast is a receiver used when casting to a google cast capable device. It is automatically used when casting from the jellyfin android client or jellyfin web client.
```

Comment:
```suggestion
Jellyfin Chromecast is a receiver app used when casting to a Google Cast capable device. It is used when casting from the Jellyfin Android client or Jellyfin web client.
```

It's probably also worth noting that the default instance of the app is hosted by Jellyfin.

Reply:
I've added a seperate "how do i use it". I did notice that the current contributing is not really up to date anymore. Since that refers to editing `jellyfin-web` if you want to host your own version. That's no longer possible since this mr: https://github.com/jellyfin/jellyfin-meta/issues/45

Though I'm uncertain how the workflow should look like now. But that's not an issue for this readme :) I've referred to the contributing.md.